app.py

```diff
@@ -100,6 +101,36 @@ def leaderboard():
 def logout():
     session.clear()
     return redirect(url_for('home'))
-    
+
+#route to request
+class requests(db.Model):
+    __tablename__= 'requests'
+    id=db.Column(db.Integer,primary_key=True)
+    content=db.Column(db.String(100),nullable=False)
+    options=db.Column(db.String(20),nullable=False)
+    timestamp = db.Column(db.DateTime, default=datetime.utcnow, nullable=False)
+
+
+@app.route('/request_route',methods=['POST','GET'])
+def request_route():
+
+    if request.method=='POST':
+        content=request.form['Content']
+        options=request.form['value']
+        new_task=requests(id=session["user_id"],content=content,options=options)
+        try:
+            db.session.add(new_task)
+            db.session.commit()
+            return redirect(url_for('feed'))
+        except Exception as e:
+            print(f"Error: {e}")
+            return redirect(url_for('feed'))    
+
+    return redirect(url_for('feed'))
+
+@app.route('/go_request')
```

Comment:
no need for two different routes, see login (or) new_post (or) redeem routes for reference.